examples/networking/xdp/xdp_drop_count.py

```diff
@@ -0,0 +1,135 @@
+#/usr/bin/env python
+#
+# xdp_drop_count.py Drop incomming packets on XDP layer and count for which
+#                   protocol type
+#
+# C code adapted from Brenden Blanco as found in kernel:samples/bpf/xdp1_kern.c
+#
+# Copyright (c) 2016 Jan Rüth
+# Licensed under the Apache License, Version 2.0 (the "License")
```

Comment:
Hm, yes I was unsure about it, as I only adapted parts in the C file, I think I have to adapt the GPL license, however the C file is actually not really part of the python file. I thought it would be ok to include the GPL note in the C part and just use the apache license (as it is used also in the other files) for the python part. would it be better to have them as separate files? Would that solve things? Do you mean drop the whole python file?
It would also be fine with me to just have the GPL license in the file, I don't know if the project requires anything.